lib/stat_tracker.rb

```diff
@@ -63,4 +64,8 @@ def percentage_visitor_wins
   def percentage_ties
     (@game_repo.games.count {|game| game.tie_game?}.to_f / @game_repo.total_games).round(2)
   end
+
+  def count_of_teams
+    @team_repo.total_teams
```

Comment:
Similar issue we encountered during refactoring- counted teams twice. Looks good here!